arrow-libs/fx/arrow-fx-coroutines/src/commonMain/kotlin/arrow/fx/coroutines/ParTupled.kt

```diff
@@ -0,0 +1,338 @@
+package arrow.fx.coroutines
+
+import arrow.core.*
+import kotlinx.coroutines.CoroutineScope
+import kotlin.coroutines.CoroutineContext
+import kotlin.coroutines.EmptyCoroutineContext
+
+/**
+ * Runs [fa], [fb] in parallel on [ctx] and combines their results as a tuple.
+ *
```

Comment:
Should we add this clarifying documentation about [context] parameter? Can we added to all functions.

```suggestion
 * Runs [fa], [fb] in parallel on [ctx] and combines their results as a tuple.
 *
 * Coroutine context is inherited from caller, additional context elements can be specified with [context] argument.
 * If the context does not have any dispatcher nor any other [ContinuationInterceptor], then [Dispatchers.Default] is used.
```